toc/rfc/rfc-draft-stemcell-release-schedule.md

```diff
@@ -0,0 +1,26 @@
+# Meta
+[meta]: #meta
+- Name: Stemcell Release Schedule
+- Start Date: 2022-06-29
+- Author(s): @thomasthal
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: https://github.com/cloudfoundry/community/pull/331
+
+
+## Summary
+This RFC proposes a well defined release schedule for the CF Stemcell to ensure the availability of OS related security patches in a timely manner.
+
+## Problem
+Almost daily, Canonical releases security patches of known vulnerabilities. As a PaaS offering provider, you want to consume those patches as fast as possible to mitigate vulnerabilities and keep your customers safe. Cloud Foundry has the huge security advantage to be able to rollout zero downtime updates and patch security vulnerabilities at any time. This potential is limited by the fact, that often there is only an "outdated" stemcell available, which does not yet contain latest patches.
+
+A second concern is that the current stemcell release strategy seems to be an undocumented, informal agreement. This makes it impossible to evaluate if the process to create a new stemcell is executed accordingly. Moreover, it is difficult to provide clear statements to customers how long the rollout of an OS patch takes or to judge if compliance requirements are met.
+
+## Proposal
+- Release a new stemcell every week on Tuesday, that contains all available ubuntu security patches 

```

Comment:
The team maintaining the linux stemcell is also responsible for cutting windows stemcells. So the linux stemcell to Patch Tuesday would create to much load on the team on these days.

Reply:
> Will shipping medium and low vulnerabilities every week actually improve our security posture much over our current every 3 weeks schedule? I'm asking because cutting stemcells does consume time, which can not be spent on improving the quality of our codebase.

IMHO it would improve security, as the combination of several low or medium vulnerabilities can become critical and faster patch cycles causes less known vulnerabilities on average. I also think that high vulnerabilities will be patched faster with this proposal - please compare my comment below. Moreover, I see the problem that depending on the scenario a medium rated vulnerability might be more severe for some use cases or parties. This would result in discussions how to rate a vulnerability, which could be avoided if you know that there will be a new stemcell in a few days.